src/batou_ext/fcio.py

```diff
@@ -71,39 +72,34 @@ def configure(self):
     def _compute_calls(self):
         assert self.project
         self.aliases = []
-        for host in self.environment.hosts.values():
+        for host in self.environment_.hosts.values():
```

Comment:
Looks like a typo for me -- _ intentionally?

Reply:
Yes,  it's a  component, hence environment is already used by batou.